packages/flutter/lib/src/widgets/color_filter.dart

```diff
@@ -11,7 +11,56 @@ import 'framework.dart';
 
 /// Applies a [ColorFilter] to its child.
 ///
+/// Causes the child to be modified using a color to tint the resulting rendering.
+/// This widget allows to generate dynamic and vibrant shades depending on the chosen blend modes.
+/// Different blending modes can modify how the color is applied,
+/// and to which parts of the image.
+/// ColorFiltered widget is different from BackdropFilter and ImageFiltered as both
```

Comment:
maybe: Use the [BackdropFilter] widget instead, if the [ColorFilter] needs to be applied onto the content beneath [child],